### DISCUSSION TITLE
Disable Login

### DISCUSSION THREAD

**Discussion by mworzala** [Unleash/unleash#3189] [Q&A]
Hi,

Is it possible to completely disable login (OSS version)?

**Comment by sighphyre** [comment]
Hi @mworzala that's a very interesting question! What are you trying to do here?

Out the box, there's no option to completely disabled login in the OSS version but you can implement your own auth handler and just reject every login attempt. There's some docs on custom auth here: https://docs.getunleash.io/reference/deploy/securing-unleash#implementing-custom-authentication

**Reply by mworzala** [reply]
Hey, thanks for the quick reply. I am actually interested in the opposite - I would like to completely remove any form of authentication. It seems like a custom auth handler could do that as well though.

**Reply by ivarconr** [reply]
You can configure unleash to use authentication type NONE, see https://docs.getunleash.io/reference/deploy/configuring-unleash#further-customization
